types/node/stream.d.ts

```diff
@@ -6,7 +6,9 @@ declare module "stream" {
     }
 
     namespace internal {
-        class Stream extends internal { }
+        class Stream extends internal {
+            constructor(opts?: ReadableOptions);
```

Comment:
I don't agree with this constructor of the legacy `Stream`. According to [code](https://github.com/nodejs/node/blob/1a552f614b4ec2b9faf0bf932ad4e6ef4fdcaf01/lib/internal/streams/legacy.js#L9-L11) options are just forwarded to `EventEmitter`.

Reply:
This seems in conflict with `readable-stream` which is kind of annoying.

Reply:
ok, seems to be no easy workaround.